app/models/sign_in/client_config.rb

```diff
@@ -89,5 +91,17 @@ def mobile_test_config
         refresh_token_duration: Constants::RefreshToken::VALIDITY_LENGTH_LONG_DAYS.days
       }
     end
+
+    def vamock_config
+      {
+        cookie_auth: true,
+        api_auth: false,
+        anti_csrf: true,
+        redirect_uri: Settings.sign_in.client_redirect_uris.web,
+        access_token_duration: Constants::AccessToken::VALIDITY_LENGTH_LONG_MINUTES.minutes,
```

Comment:
It probably doesn't matter a huge amount, but I wonder if we should make these durations the 'short' versions of these to match the other `vaweb` implementation

Reply:
I wondered about that, I assumed that the intent was to have mocked sessions be long so that people who don't care about switching users but just want to always be logged in on their preferred account don't have to interact with authentication at the start of each day. But that was my assumed use case from reading the issue ticket - happy to make these in line with normal `vaweb` auth.